app/build.gradle.kts

```diff
@@ -81,6 +81,12 @@ dependencies {
 
     implementation(libs.timber)
 
+    implementation(libs.okio)
+    implementation(libs.moshi)
+    implementation(libs.moshi.kotlin)
```

Comment:
This should not be requirement - fix https://github.com/hossain-khan/kgeo-device-timeline/issues/21